### ISSUE TITLE
prefer-string-slice bad example

### ISSUE THREAD

**Issue by lagden** [sindresorhus/eslint-plugin-unicorn#480]
A better example may help. Because it's not 1:1

```js
const _hex = '0054006100640061007300680069'

_hex.substr(0, 4) // "0054"
_hex.substr(4, 4) // "0061"

_hex.slice(0, 4) // "0054"
_hex.slice(4, 4) // ""

// fixed
_hex.slice(4, 8) // "0061"
```

**Comment by sindresorhus** [comment]
Why is the example bad?

**Comment by fregante** [comment]
Because apparently it suggests that `slice(4,4)` is the same as `substr(4,4)` but in fact the former means nothing as the result is always `''`

**Comment by fregante** [comment]
The actual example uses `1, 3` as indexes; the slice one should be `1, 4` to have the same result afaik 

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/prefer-string-slice.md
